hnn_core/tests/test_params.py

```diff
@@ -1,15 +1,61 @@
 # Authors: Mainak Jas <mainakjas@gmail.com>
+#          Blake Caldwell <blake_caldwell@brown.edu>
 
 import os.path as op
+import json
+
+import pytest
+from mne.utils import _fetch_file
 
 import hnn_core
-from hnn_core import Params
+from hnn_core import read_params, Params
 
 
 def test_params():
     """Test params object."""
     hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
     params_fname = op.join(hnn_core_root, 'param', 'default.json')
-    params = Params(params_fname)
+    params = read_params(params_fname)
     print(params)
     print(params['L2Pyr*'])
+
+
+def test_legacy_params():
+    """Test reading legacy .param file."""
+    hnn_core_root = op.join(op.dirname(hnn_core.__file__), '..')
```

Comment:
@blakecaldwell I merged the data PR. Can you make a push to this PR to activate Travis again?

Perhaps you can move this line outside the function so it can be reused by all of them